src/main/java/bio/terra/service/filedata/google/firestore/FireStoreUtils.java

```diff
@@ -43,11 +45,14 @@ public class FireStoreUtils {
   // Firestore limits batches to 500
   public static final int MAX_FIRESTORE_BATCH_SIZE = 500;
 
-  private ConfigurationService configurationService;
+  private final ConfigurationService configurationService;
+  private final long futureTimoutSeconds;
```

Comment:
Maybe worth a docstring as to why we want this, since this has been such a tricky-to-debug issue.